src/send/SendSelectRecipient.tsx

```diff
@@ -81,20 +81,25 @@ function GetStartedSection() {
 
   return (
     <View style={getStartedStyles.container} testID={'SelectRecipient/GetStarted'}>
-      <Text style={getStartedStyles.subtitle}>{t('sendSelectRecipient.getStarted.subtitle')}</Text>
```

Comment:
This order and naming threw me through a loop for a bit, `headline` and `title` are probably more appropriate.

Reply:
I think this can be fixed in a separate PR